src/napari/_qt/widgets/qt_viewer_buttons.py

```diff
@@ -557,23 +557,28 @@ def _open_grid_popup(self):
         grid_width = QtSpinBox(popup)
         grid_height = QtSpinBox(popup)
         grid_spacing = QDoubleSpinBox(popup)
+        grid_border_width = QtSpinBox(popup)
```

Comment:
This should be QDouble, if the fractional setting is to work from the tooltip.
Edit: But maybe it's the tooltip that is lying, because I tested this locally and still doesn't work :P